test/evergreen.yml

```diff
@@ -2067,6 +2067,35 @@ tasks:
             cp ../../../bench/wtperf/split_heavy.wtperf .
             ./wtperf -O ./split_heavy.wtperf -o verbose=2
 
+  - name: static-wt-build-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
```

Comment:
I believe the `CC='eval "g++ -static"'` setting is essential for this test. A `configure_env_vars` var overloading will do?

Reply:
> I believe the `CC='eval "g++ -static"'` setting is essential for this test. A `configure_env_vars` var overloading will do?

`CC='eval "g++ -static"'` will be set to rebuild static "wt" util.